### PR TITLE
Add Image Signing From K8s RelEng

### PR DESCRIPTION
**Pull request by puerco**
This commit adds to the project ideas a proposal from Kubernetes Release Engineering. We are seeking help to fix image signatures in our image promotion process.

/hold 
For final RelEng review
/cc @mehabhalodiya

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
+1, I'm in support of this.

cc @nate-double-u @aliok

**Comment by puerco** [comment]
@nate-double-u @aliok is it possible to add more than one two mentors? This project will likely require more than the two of us.

**Comment by aliok** [comment]
@puerco add as many as you like!
We're ready to merge this one, but I am waiting for you to update the PR with new mentors.

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md
Comment (aliok) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:164]:
```suggestion
- Upstream Issues:
  - https://github.com/kubernetes/registry.k8s.io/issues/187
  - https://github.com/kubernetes/release/issues/2962
```

**Comment by mehabhalodiya** [comment]
@puerco Please add @xmudrii to the mentor's list. He will be going to help as much as he can without a doubt and he also confirmed that he'll have plenty of time capacity for image promoter work. 🙂
Thank you!

**Comment by puerco** [comment]
OK, I've added @xmudrii and @jeremyrickard to the list of mentors

**Comment by jeremyrickard** [comment]
+1 from me

**Comment by puerco** [comment]
OK, I stupidly added `/hold` to note that we were still working on it not realizing we don't have prow here, sorry about that!@ :upside_down_face: 

@aliok we're ready to go with this one, we discussed the proposal and plan on the SIG Release meeting on May 7th and we're on.

**Comment by nate-double-u** [comment]
Since this is a proposal for a Kubernetes project, I think we'll need the sig-contribex leads approval as well. Usually we'd require a project maintainer's approval, but things are a touch different with K8s projects and subprojects. @kaslin & @palnabarun, if you could both take a look

**Comment by justaugustus** [comment]
Another SIG Release Chair +1 from me!

**Review by aliok — APPROVED** [review]

**Comment by aliok** [comment]
Thanks folks!

**Comment by kaslin** [comment]
Oops, we never did respond here. I don't think SIG ContribEx approval is explicitly required, but as long as the folks running it believe it makes sense and are excited to do it, we're happy to offer our approval!

**Comment by kaslin** [comment]
We do also appreciate being involved so we can try to keep track of mentoring efforts going on in the project, so maybe requiring SIG ContribEx approval is the way to achieve that.

**Pull request merged**
